### PR TITLE
🐛 Collect debugging information as expected error

### PR DESCRIPTION
**Pull request by zhouyx**

### CODE REVIEW

File: extensions/amp-analytics/0.1/requests.js [hunk 1]
```diff
@@ -570,7 +570,7 @@ function reportErrorTemp(url, info) {
     const elementWidth = Number(elementSize[0]);
     const elementHeight = Number(elementSize[1]);
     if (elementWidth == 0 || elementHeight == 0) {
-      dev().error(
+      dev().expectedError(
```
Comment (zhouyx) [inline extensions/amp-analytics/0.1/requests.js:573]: Question: How to tell if we can aggregate the logs? By collecting static error messages?

**Comment by zhouyx** [comment]
I'm hoping to keep this to verify my fix which I'm still working on. Any idea on reducing the frequency?

**Comment by zhouyx** [comment]
What are other ways to collect this data that allow for aggregation? I can switch to that, maybe in a following PR (not this cherrypick one)

**Comment by rcebulko** [comment]
> I'm hoping to keep this to verify my fix which I'm still working on. Any idea on reducing the frequency?

`user().expectedError()` throttles by another factor of 10

**Comment by rcebulko** [comment]
> What are other ways to collect this data that allow for aggregation? I can switch to that, maybe in a following PR (not this cherrypick one)

Stackdriver and the error reporting infrastructure is not designed for use in collecting debugging analytics like this. As far as I know, we really don't have any tooling to support this level of fine-grained data collection (partially for privacy reasons I believe)

**Comment by zhouyx** [comment]
I see. Thanks! 

Do you want me to switch to `user(). expectedError()` here? Is the throttling the only difference between `dev().expectedError()` and `user().expectedError()`? 
Since this is not a user error, should I throttle it myself? I can throttle it even further.

**Comment by rcebulko** [comment]
As long as it's under Expected Errors, it's a bit less of a big deal. Maybe throttling manually by another factor of 10 would be reasonable, but not strictly necessary as Expected Errors are looked at less so the noise is less impactful

**Comment by zhouyx** [comment]
FYI: Throttle by another 10%

**Comment by zhouyx** [comment]
@micajuine-ho to approve the bundle size change.

File: extensions/amp-analytics/0.1/requests.js [hunk 2]
```diff
@@ -585,6 +589,9 @@ function reportErrorTemp(url, info) {
     const viewportHeight = Number(viewportSize[1]);
 
     if (viewportWidth == 0 || viewportHeight == 0) {
+      if (Math.random() > REPORTING_THRESHOLD) {
```
Comment (rcebulko) [inline extensions/amp-analytics/0.1/requests.js:592]: Suggestion: this could go into the if block above instead

Comment (zhouyx) [reply extensions/amp-analytics/0.1/requests.js:592]: 😄  I'll leave it as it is since this will go away eventually

Comment (rcebulko) [reply extensions/amp-analytics/0.1/requests.js:592]: I figured as much haha

**Pull request merged**